src/kernels/conv3x3.s

```diff
@@ -729,32 +681,49 @@ miopenGcnAsmConv3x3U:
       s_addc_u32 s[weights_ptr+1], s[weights_ptr+1], 0
   .endif
 
+.macro s_mul_u64_u32 dst_lo, dst_hi, src0, src1, vtmp0, vtmp1
```

Comment:
[Recommendation]
```suggestion
.macro s_mul_u64_u32 sdst_lo, sdst_hi, ssrc0, ssrc1, vtmp0, vtmp1
```

Reply:
Thanks for the recommendation, fixed in bd0f6aa